spec/helpers/claims_helper_spec.rb

```diff
@@ -177,4 +178,28 @@ def current_user
       end
     end
   end
+
+  describe '#fee_shared_headings(claim, scope)' do
```

Comment:
We don't typically name the args in the rspec describe

Reply:
Thanks. That was a copy-paste and I forgot to edit it.